### PR TITLE
add docker setup

### PR DESCRIPTION
**Pull request by heiderich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,22 @@ All of the options below are optional except for the `HOST[:PORT]` URI of your E
...
+A docker image can be build using
+
+```sh
+docker build -t elasticsearch-primary-balancer .
+```
+
+Then the balancer can be run using
+
+```sh
+docker run --rm elasticsearch-primary-balancer:latest <options>
```
Comment (hallh) [inline README.md:72]: Due to the issue you found in #9, this command will fail with the `clearLine()` crash. If you change the run command to `docker run -it ...` then docker will run it interactivaly and assign a pseudo-TTY that prevents the crash.

Besides that, I generally prefer to include the `-it` flags for running dockers manually as it also gives you the option to interrupt the execution with Ctrl-C, so include that, and the PR is golden.

**Review by hallh — CHANGES_REQUESTED** [review]
I've one request regarding the docker run command, but otherwise looking great. Thanks for contributing this!

File: README.md (hunk 1 above)
Comment (heiderich) [reply README.md]: I generally also use `-it`, but forgot to add it here. I added it in 50956a9. With c584586 both versions should work.

**Review by hallh — APPROVED** [review]

**Comment by hallh** [comment]
Good stuff, merging this 😎

**Pull request merged**